website/docs/validation/seccompv2.md

```diff
@@ -0,0 +1,496 @@
+---
+id: seccompv2
+title: Seccomp V2
+---
+
+# Seccomp V2
+
+## Description
+Controls the seccomp profile used by containers. Corresponds to the `seccomp.security.alpha.kubernetes.io/allowedProfileNames` annotation on a PodSecurityPolicy. For more information, see https://kubernetes.io/docs/concepts/policy/pod-security-policy/#seccompv2
```

Comment:
what's different between non-v2 and v2?

Reply:
looks like difference is the annotation. can we document it here?

Reply:
Updated description.